docs/tutorials/multimodal/convert_data_to_coco_format.md

```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
+
+:label:`convert_data_to_coco_format`
```

Comment:
Here I suggest using sec_automm_detection_convert_to_coco.

Reply:
Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0